### ISSUE TITLE
UnusedPrivateElements don't detect calls on clones

### ISSUE THREAD

**Issue by amorimjuliana** [slevomat/coding-standard#642]
I'm using the sniff `SlevomatCodingStandard.Classes.UnusedPrivateElements` and I've noticed that it cannot identify private method calls on clones.

Example:

```php
public function foo() : void
{
    $clone = clone $this;
    $clone->writeFoo();
}

private function writeFoo() : void
{
}

public function bar() : void
{
    $this->writeBar();
}

private function writeBar() : void
{
}
```

Output:
```
ERROR | Class FooBar contains unused private method writeFoo().
```

**Comment by kukulich** [comment]
Yes, it's too difficult to solve every situation with PHPCS. For special cases it's better to use `phpcs:disable` comment.

**Comment by amorimjuliana** [comment]
Thanks anyway 😊
